students/TimPauley/session03/add_customer_test.py

```diff
@@ -0,0 +1,27 @@
+#Tim Pauley
+#Assignment 03
+#Jan 22 2019
+
+from customer_model import Customer
+
+def add_customer(
+		customer_id
+		,name
+		,last_name
+		,home_address
+		,phone_number
+		,email_address
+		,status
+		,credit_limit
+		):
+	'new_customer' = customer_id
+	,'name' = name
+	,'last_name' = last_name
+	,'home_address' = home_address
+	,'phone_number' = phone_number
+	,'email_address' = email_address
+	,'status' = status
+	,'credit_limit' = credit_limit
+new_customer.save()
```

Comment:
It looks like you are assigning strings to your variables. I don't think this is what you meant to do. Remember you have to use the create function, so the first line in this function will be something like 
```
new_customer = Customer.create(customer_id=customer_id,
    first_name=first_name,
    last_name=last_name,
    ...)
```
It worked out a bit strange that all of our variable names are the same as the column names for our model, but this is not unusual.